lib/everypolitician/popolo/collection.rb

```diff
@@ -13,9 +13,11 @@ def self.entity_class(entity = nil)
       end
 
       def initialize(documents, popolo = nil)
-        @documents = documents ? documents.map { |p| self.class.entity_class.new(p, popolo) } : []
+        @event_class = {}
```

Comment:
I think then we're just going to push it back up again though as we use it in other Collection classes.